indra/newview/llfriendcard.cpp

```diff
@@ -534,20 +534,7 @@ void LLFriendCardsManager::syncFriendsFolder()
 	// Create own calling card if it was not found in Friends/All folder
 	if (!collector.isAgentCallingCardFound())
 	{
-		LLAvatarName av_name;
-		LLAvatarNameCache::get( gAgentID, &av_name );
```

Comment:
Name cache may not have retrieved the name yet, which results in an empty av_name.